tests/unit/test_zowe_core.py

```diff
@@ -166,22 +166,38 @@ class TestZosmfProfileManager(TestCase):
     def setUp(self):
         """Setup fixtures for ZosmfProfile class."""
         # setup pyfakefs
+        self.session_arguments = {"verify": False} 
```

Comment:
@t1m0thyj I did changed the verify flag to `false` for verification, please review that :relaxed: